css/main.css

```diff
@@ -0,0 +1,402 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
```

Comment:
Your portfolio is looking great 🔥 . In order to meet the requirements, please ensure that your code is DRY. For example, instead of repeating `display: flex`, so can create a class for elements that use flexbox. That way, you avoid repeating the same lines of code, making it look more professional and up to standard. You can find more info about best practices on this link: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md